inji-verify-sdk/README.md

```
@@ -65,20 +65,63 @@ function MyApp() {
 
 ## Response Received
 
-When verification is completed, the response received is as below:
+When verification is completed, the response received is based on summariseResult attribute.This attribute will decide the format of the response from SDK.
 
+#### QRCodeVerification
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)